assets/scss/templates/_header.scss

```diff
@@ -50,6 +49,10 @@
         display: block;
         height: 50px;
       }
+
+      span {
+        line-height: 32px;
```

Comment:
why not 30 here? to make it a round number

Reply:
changed

Reply:
If 30 is too tight as you've mentioned, leave it that way please.